pxr/usd/usdPhysics/parseDesc.h

```diff
@@ -0,0 +1,896 @@
+//
+// Copyright 2024 Pixar
+//
+// Licensed under the terms set forth in the LICENSE.txt file available at
+// https://openusd.org/license.
+//
+#ifndef USDPHYSICS_PARSE_DESC_H
+#define USDPHYSICS_PARSE_DESC_H
+
+/// \file usdPhysics/parseDesc.h
+
+#include "pxr/pxr.h"
+#include "pxr/usd/usd/prim.h"
+#include "pxr/usd/usd/collectionMembershipQuery.h"
+
+#include "pxr/base/gf/vec3f.h"
+#include "pxr/base/gf/quatf.h"
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+// -------------------------------------------------------------------------- //
+// PHYSICSPARSEDESC                                                           //
+// -------------------------------------------------------------------------- //
+
+/// Sentinel value for flt max compare
+const float usdPhysicsSentinelLimit = 0.5e38f;
```

Comment:
For consistency with other globals in OpenUSD, should this be `UsdPhysicsSentinelLimit`?

Could/should it be `constexpr`?